src/components/Details.css

```diff
@@ -0,0 +1,51 @@
+
+.weather-card-content {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  margin: 12px 0;
+}
+
+.card2 {
+  background-color: #fd2;
+  width: 98%;
+  margin: 2px 0;
+  box-shadow: 0 2px 4px rgba(0, 0, 0, 0.1);
+  border-radius: 5px;
+}
+
+.card img {
+  max-width: 100%;
+}
+
+.center-container {
+  text-align: center;
+}
+
+.intro2 {
+  font-family: 'Raleway', sans-serif;
+  color: whitesmoke;
+  font-size: 16px;
+}
+
+.loading-spinner {
+  border: 4px solid rgba(0, 0, 0, 0.3);
+  border-top: 4px solid black;
+  border-radius: 50%;
+  width: 24px;
+  height: 24px;
+  animation: spin 1s linear infinite;
+  position: relative;
+  left: 48%;
+}
+
+@keyframes spin {
+  0% { transform: rotate(0deg); }
+  100% { transform: rotate(360deg); }
+}
```

Comment:
- [ ] Good job, the loading spinner looks great :+1:

Reply:
Appreciated 💯 .